source/Sashimi.AzureCloudService/AssemblyInfo.cs

```diff
@@ -0,0 +1,6 @@
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("Sashimi.AzureCloudService.Tests")]
```

Comment:
I tried putting this in the project file, but somehow it didn't work:
https://blog.sanderaernouts.com/make-internals-visible-with-new-csproj-format

Reply:
it worked for me!